src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -57,5 +83,142 @@ public void SendAsync_SimpleGet_Success()
                     "Expected cancellation requested on token attached to exception.");
             }
         }
+
+        [Fact]
+        public void GetAsyncAutomaticDecompression()
+        {
+            var handler = new HttpClientHandler();
+            if (!handler.SupportsAutomaticDecompression)
```

Comment:
libcurl supports it. I was only worried about windows version on our CI tests. I thought this should not break a checkin. I will change this to an assert in the next iteration.